server/routes/api/users.js

```diff
@@ -48,3 +49,106 @@ router.put(
       })
    }
 );
+
+//////////////////////////////
+// Skills routes under here //
+//////////////////////////////
+
+// Connects user to skill
+// Authorization required
+// If skill DNE, create it
+
+// @route   POST /api/users/newSkill
+// @desc    Connects user to skill, if skill DNE, create it
+// @body    { skillName }
+// @access  Authorized
+router.post(
+   "/newSkill",
+   passport.authenticate("jwt", { session: false }),
+   async (req, res) => {
+      const user = req.user
+      const { skillName } = req.body
+      try {
+         // Look for skill
+         let skill = await Skill.findOne({
+            where: {
+               skill: skillName
+            }
+         })
+
+         let createUserSkill = (user, skill) => {
+            // Create UserSkill
+            let userId = user.id
+            let skillId = skill.id
+            console.log(`User '${user.name}' has added skill '${skill.skill}'`)
+            UserSkill.create({ userId, skillId }).then(userSkill => res.send(userSkill))
```

Comment:
You could also use this method of adding a skill (gota double check syntax)
User.findByPk(user).then(user => user.addSkill(skill)) 

This is a sqeulize thing, and it helps with using the associations